jetbrains-ultimate/src-201+/software/aws/toolkits/jetbrains/services/rds/RdsEngine.kt

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.rds
+
+import icons.AwsIcons
+import software.aws.toolkits.resources.message
+import javax.swing.Icon
+
+enum class RdsEngine(val engine: String, val displayName: String, val icon: Icon, val resourceType: String = engine) {
+    MySql("mysql", message("rds.mysql"), AwsIcons.Resources.Rds.MYSQL) {
+        override fun iamConnectionStringUrl(url: String) = "jdbc:$jdbcMysql://$url/"
+    },
+    Postgres("postgres", message("rds.postgres"), AwsIcons.Resources.Rds.POSTGRES) {
+        override fun iamConnectionStringUrl(url: String) = "jdbc:$jdbcPostgres://$url/"
+        override fun iamUsernameHook(username: String) = username.toLowerCase()
```

Comment:
Can you copy the comment back to here?

Reply:
> Also, a thought, maybe we should only use the [] to mark it as Aurora, the icon should be enough?

So you're saying 'plain' DB's don't have a status, but the aurora ones do? I can get on board with that.

Reply:
Yep, exactly

Reply:
done